components/suggestion/search_date_suggestion/search_date_suggestion.tsx

```diff
@@ -3,11 +3,11 @@
 
 import React from 'react';
 
-import DayPicker from 'react-day-picker';
+import DayPicker, {Modifier} from 'react-day-picker';
```

Comment:
should this be `DayModifiers` ?

Reply:
@M-ZubairAhmed, the type of modifier is
![Screenshot from 2022-10-14 10-01-40](https://user-images.githubusercontent.com/89139362/195763081-18a76daa-8001-4539-99db-167e8daf0b00.png)
So it gives an error **Type '{ today: DayModifiers; } | undefined' is not assignable to type 'Partial<Modifiers> | undefined'.**